synapse/lib/trigger.py

```diff
@@ -358,3 +306,69 @@ def get(self, iden):
         if rule is None:
             raise s_exc.NoSuchIden(iden=iden)
         return dataclasses.asdict(rule)
+
+class TriggerStorage:
```

Comment:
We may want to consider creating separate view metadata storage ( maybe within top layer? )